app/assets/stylesheets/pages/_spots.scss

```diff
@@ -5,6 +5,15 @@
 .spots__message-wrapper {
   @include wrapper();
   @include spacer();
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.spots__message {
+  text-align: center;
+  margin: $spacing-5 0px;
 }
```

Comment:
You'll see it aligned to the left now. I changed it just until I get confirmation from Laura. I think she told me at one point that she wanted everything to go left–even though the design doesn't reflect it. Just for your information, in case you wonder what happened. :-)